blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -430,6 +450,26 @@ private MetricData getStringMetricDataForRange(Locator locator, Range range, Gra
         return new MetricData(points, getUnitString(locator), MetricData.Type.STRING);
     }
 
+    private MetricData getEnumMetricDataForRange(final Locator locator, Range range, Granularity gran, RollupType rollupType, DataType dataType) {
```

Comment:
Hey George! I've made another callable to get the points so as to remove redundancy.
Yes, type and rollup type are two different things.
Also I've repaired the exceptions.